### PR TITLE
[16.0][MIG] product_pricelist_item_list_view

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration from v9. Reusing default pricelist item views to reduce to the minimum the code needed: one menu item and minor extension of search and form views.

@ForgeFlow

### CODE REVIEW

File: product_pricelist_item_list_view/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "category": "Sales Management",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Agile Business Group, ForgeFlow, Odoo Community Association (OCA)",
+    "maintainers": ["LoisRForgeFlow"],
+    "development_status": "Production/Stable",
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "product",
```
Comment (MiquelRForgeFlow) [inline product_pricelist_item_list_view/__manifest__.py]:
```suggestion
        "sale",
```

**Comment by AaronHForgeFlow** [comment]
Do you know why runboat is not available? Tried to reset it but it is still red

File: product_pricelist_item_list_view/__manifest__.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply product_pricelist_item_list_view/__manifest__.py]: Indeed, it is fixed now.

thanks!

**Comment by LoisRForgeFlow** [comment]
> Do you know why runboat is not available? Tried to reset it but it is still red

No idea :confused:. I have pushed again, let's see this time

**Review by AaronHForgeFlow — APPROVED** [review]
Now I can access :)

LGTM

**Pull request merged**